openedx/core/djangoapps/programs/models.py

```diff
@@ -132,3 +137,10 @@ def is_certification_enabled(self):
         certificates for Program completion.
         """
         return self.enabled and self.enable_certification
+
+    @property
+    def show_xseries_ad(self):
```

Comment:
I'm thinking specifically about open source. Programs aren't yet officially supported for open source use, but we can make that process easier by designing more generically, earlier. @jimabramson thoughts?